### DISCUSSION TITLE
help to make Master Detail Grid in Center location

### DISCUSSION THREAD

**Discussion by Emeronerp1** [serenity-is/Serenity#6669] [Help]
Hi all,

I'm not good in front end , i would like to know how to make the master detail grid aligned to center of the dialog like i want to remove the grid caption area and make center of the form.
![image](https://user-images.githubusercontent.com/45841416/214776922-227007ce-0085-4243-9e68-8eeb1e161983.png)

in the item list field i can see caption field which is taking the space, i want to remove that and make the grid fit in the whole area

how to achieve this,

also if some body can guide me to hide the top nav bar whenever certain forms are opening like transaction forms , if i open transaction kind of forms i want to hide the top navbar and when ever i click on dashboard then only required to see that one, how to achieve this.

i am very poor in front end my background was in winform application development.

Thanks in advance

**Comment by reach2rv** [comment]
In your dialog.ts

```
editor.element.siblings('label').hide();

```

Where editor is your grid field
